charts/render.py

```diff
@@ -114,25 +120,45 @@ def keep_doc(doc):
 
 
 def main():
-    (name, namespace, values, path) = sys.argv[1:]
+    parser = argparse.ArgumentParser()
+    parser.add_argument('name', help="Denotes the name of the chart")
+    parser.add_argument('namespace', help="Metalk8s namespace to deploy unto")
+    parser.add_argument('values', help="Our custom chart values")
+    parser.add_argument(
+        '--service-config',
+        action='append',
+        nargs=2,
+        dest="service_config",
+        help="Ex: --service-config service_name service_configmap_name"
+    )
+    parser.add_argument('path')
```

Comment:
Let's add some help as well:

```suggestion
    parser.add_argument('path', help="Path to the chart directory")
```